cloudbridge/cloud/providers/gce/services.py

```diff
@@ -797,14 +799,28 @@ def create(self, network, cidr_block, name=None, zone=None):
                 return None
             subnets = self.list(network, region)
             for subnet in subnets:
-                cb.log.warning('subnet ID: %s', subnet.id)
                 if subnet.id == response['targetId']:
                     return subnet
         except:
             return None
 
     def get_or_create_default(self, zone=None):
-        raise NotImplementedError('To be implemented')
+        """
+        Every GCP project comes with a default auto mode VPC network. An auto
```

Comment:
I think this is true for every *new* GCP project, but is not true for old projects.

Reply:
The same situation happens on AWS so if a provider-default does not exist, we create a CloudBridge-default: https://github.com/gvlproject/cloudbridge/blob/master/cloudbridge/cloud/providers/aws/services.py#L884

Reply:
Done: fixed in #74.